<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareSettings</class>
 <widget class="QDialog" name="HardwareSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hardware Settings</string>
  </property>
  <widget class="QLabel" name="DefaultSettingsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>490</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Default Hardware Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UserSettingsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>490</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>User Hardware Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DefaultHardwareSettingsFilename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>490</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="UserSettingsBrowseButton">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>140</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserSettingsLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>305</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadUserSettingsButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>UserSettingsLineEdit</tabstop>
  <tabstop>LoadUserSettingsButton</tabstop>
  <tabstop>UserSettingsBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>UserSettingsBrowseButton</sender>
   <signal>pressed()</signal>
   <receiver>HardwareSettings</receiver>
   <slot>FindUserSettingsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadUserSettingsButton</sender>
   <signal>pressed()</signal>
   <receiver>HardwareSettings</receiver>
   <slot>LoadUserSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UserSettingsLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>HardwareSettings</receiver>
   <slot>UserSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UserSettingsLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>HardwareSettings</receiver>
   <slot>ActivateLoadButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>HardwareSettings</receiver>
   <slot>QuitApplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
